pkg/upload/android-ndk.go

```diff
@@ -155,6 +136,25 @@ func ProcessAndroidNDK(apiKey string, applicationId string, androidNdkRoot strin
 			if strings.HasSuffix(file, ".so.sym") {
 				symbolFileList = append(symbolFileList, file)
 			} else if filepath.Ext(file) == ".so" {
+				// Check NDK path is set
+				androidNdkRoot, err = android.GetAndroidNDKRoot(androidNdkRoot)
+
+				if err != nil {
+					return err
+				}
+
+				log.Info("Android NDK Path: " + androidNdkRoot)
+
+				// Find objcopy within NDK path
+				log.Info("Locating objcopy within Android NDK path")
+
+				objCopyPath, err := android.BuildObjcopyPath(androidNdkRoot)
+
+				if err != nil {
+					return err
+				}
+
+				log.Info("Objcopy Path: " + objCopyPath)
```

Comment:
This logging feels a bit over-the-top, can you put this block inside an if-statement that only executes if `objCopyPath` is empty (and declare that variable above)?

Then condense down to a single Info message - "Located objcopy within Android NDK path: {path}".